### PR TITLE
Finished bottom half of RecordingPage UI

### PR DESCRIPTION
**Pull request by Axelh04**
**Description:**

- Added styling to all elements, made sure they worked with each other. Can alter styling later if needed. 
- Note: The note variable is being passed down to similar sounds, but is not used (this is on purpose). Will be used in the future, but I am going to change its use in a later pull request.

**Next steps:**

- Landing/Profile UI
- Add loading state for Recording Page
- Navbar TouchUps
- All page touch ups

Test Plan:
https://www.loom.com/share/e6b137a841b44368a18b99a191547fc8?sid=a53becd1-6d5f-4ddd-8487-45f4e56e61e8

### CODE REVIEW

File: frontend/src/RecordingPageComp/PlaybackContainerComp.jsx [hunk 1]
```diff
@@ -264,22 +265,36 @@ function PlaybackContainer({
...
+            </span>
+          </div>
+        ) : (
+          <div> No Instrument Selected.</div>
+        )}
+      </div>
+      <button id="play-button" onClick={startPlayback}>
+        <img
+          alt="play-icon"
+          src="https://www.pngall.com/wp-content/uploads/5/Black-Play-Button-PNG-Free-Download.png"
```
Comment (nhsoukayna) [inline frontend/src/RecordingPageComp/PlaybackContainerComp.jsx:288]: I would discourage you from using external assets like this. Since you don't have control over it it could easily break your application with no change on your side.
It would be better if you download these images and store them locally in your repo.

File: frontend/src/RecordingPageComp/SimilarSounds.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-function SimilarSounds({ note, playbackDuration }) {
+function SimilarSounds({ playbackDuration }) {
   const duration = Math.floor(playbackDuration);
   const [soundList, setSoundList] = useState([]);
 
   const fetchSounds = async () => {
     try {
       const API_KEY = import.meta.env.VITE_APP_API_KEY;
       const response = await fetch(
-        `https://freesound.org/apiv2/search/text/?token=${API_KEY}&filter=duration:${duration}%20ac_note_midi:${note}&fields=name,previews`
```
Comment (nhsoukayna) [inline frontend/src/RecordingPageComp/SimilarSounds.jsx:13]: just curious why did you remove note?

Comment (kfullen) [inline frontend/src/RecordingPageComp/SimilarSounds.jsx:13]: I'd suggest adding error handling for your API request here in case the request fails. Make sure to include error handling for all network requests to handle any errors gracefully! It's good to always consider edge cases and failure cases when building your features and build in error handling to from the start
